plone/app/layout/viewlets/social.py

```diff
@@ -96,7 +98,7 @@ def _get_tags(self):
                 scales = self.context.restrictedTraverse('@@images', None)
                 if scales:
                     try:
-                        image = scales.scale('image', scale='great')
+                        image = scales.scale('image', scale=SOCIAL_IMAGE_SCALE)
                     except Exception as e:
                         logger.exception(e)
```

Comment:
Just came accross an error, because `logger` is defined nowhere when an exception happens here. I suggest fixing this within this PR ...

Reply:
👍